crates/bevy_math/src/cubic_splines.rs

```diff
@@ -123,10 +123,10 @@ impl<P: Point> CubicGenerator<P> for CubicBezier<P> {
 /// let hermite = Hermite::new(points, tangents).to_curve();
```

Comment:
should be renamed here too?

Reply:
I was hoping the doc tests would catch any of these misses.

Reply:
Good news: they did.